### PR TITLE
Add dependency Text::Markdown

### PR DESCRIPTION
**Pull request by perlpunk**
This will be needed for the HTML email generation

Issue: https://progress.opensuse.org/issues/111215

### CODE REVIEW

**Review by okurz — COMMENTED** [review]
hm, don't we need the according update in the spec file then?

**Comment by perlpunk** [comment]
Right... added now.
That actually reveals that we have no test here to check that.

**Comment by perlpunk** [comment]
I added a test

File: xt/01-make-update-deps.t [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+chdir $build_dir;
+my @out = qx{$make_cmd};
+my $rc = $?;
+die "Could not run $make_cmd: rc=$rc, out: @out" if $rc;
+
+my @status = grep { not m/^\?/ } qx{git -C "$Bin/.." status --porcelain};
+ok(!@status, "No changed files after '$make_cmd'") or diag @status;
+
+done_testing;
+
```
Comment (okurz) [inline xt/01-make-update-deps.t:28]: That's a copy-paste from os-autoinst. I would favor to not duplicate. How about putting the test into os-autoinst-common and call it from there? As alternative we could symlink from external into xt/

**Review by okurz — CHANGES_REQUESTED** [review]

File: xt/01-make-update-deps.t (hunk 1 above)
Comment (perlpunk) [reply xt/01-make-update-deps.t:28]: ok, but I will split it out of this PR so we can make actual progress.

**Review by Martchus — APPROVED** [review]

**Review by okurz — APPROVED** [review]

**Pull request merged**